cli/types/cypress.d.ts

```diff
@@ -5730,6 +5774,8 @@ declare namespace Cypress {
     message: any
     /** Set to false if you want to control the finishing of the command in the log yourself */
     autoEnd: boolean
+    /** Set to false if you want to control the finishing of the command in the log yourself */
```

Comment:
This comment is not accurate.

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/21137/commits/76e51782053173ff93f81edabb68c88e55ea285c